### ISSUE TITLE
Missing nanoget arg

### ISSUE THREAD

**Issue by Adamtaranto** [wdecoster/nanofilt#2]
Nanofilt dies when trying to process albacore summary files via nanoget. processSummary(summaryfile, readtype) requires and additional arg for readtype, either "1D" or "2D".

**Line 50 of NanoFilt.py:**
`data = {entry[0]: entry[1] for entry in processSummary(args.summary)[["readIDs", "quals"]].itertuples(index=False)}`

**NanoFilt appears to work with this edit:**
`data = {entry[0]: entry[1] for entry in processSummary(args.summary, "1D")[["readIDs", "quals"]].itertuples(index=False)}`

Probably useful to have readtype as a command line arg in case anyone still has 2D data?

**Comment by wdecoster** [comment]
Thanks for reporting this! 

Right, that's a silly mistake on my side. I adapted nanoget processSummary() for NanoPlot to enable users to select the read type of interest but forgot to adapt NanoFilt which uses the same function. Sorry for this!

This has now been fixed in NanoFilt 1.1.4. There is an optional `--readtype` command line argument to set the read type, the default is 1D. The new version is available on pip. I'll have to figure out how to update build on bioconda.

Cheers,  
Wouter
